examples/lb3-application/README.md

```diff
@@ -337,13 +337,205 @@ $ npm start
 Load [http://localhost:3000/](http://localhost:3000/) on your browser. This will
 load the Express app, with mounted LB3 and LB4 applications.
 
-### Need help?
+## Running LB3 tests from LB4
+
+You can run tests in an LoopBack 3 application from the LoopBack 4 application
+it mounted on with command `npm test`.
+
+We want the LoopBack 3 tests to use the LoopBack 4 server rather than the
+LoopBack 3 application. The following guide shows how to run
+
+- acceptance-level tests making HTTP calls to invoke application logic. e.g.
+  `POST /users/login`
+- integration-level tests that are using JS API to call application logic. e.g.
+  `MyModel.create()`
+
+### Adding LB3 Test Path in Command
+
+In order to run LoopBack 3's tests from their current folder, add LB3 tests'
+path to `test` entry in package.json:
+
+- `"test": "lb-mocha \"dist/**tests**/\*_/_.js\" \"lb3app/test/\*.js\""`
+
+In this case, the test folder is
+[`/lb3app/test`](https://github.com/strongloop/loopback-next/tree/spike/lb3test/examples/lb3-application/lb3app/test)
+from the root of the LoopBack 4 project.
+
+This will run LoopBack 4 tests first then LoopBack 3 tests.
+
+_To emphasize the setup steps and separate them from the test case details, all
+the comprehensive test code are extracted into function `runTests`._
+
+### Running Acceptance Tests
+
+First, move any LoopBack 3 test dependencies to `package.json`'s devDependencies
+and run:
+
+```sh
+npm install
+```
+
+In your test file:
+
+1. When launch the Express server
```

Comment:
As someone who's interested in running my LB3 app tests, I'd be confused why I need Express server.

Reply:
@hacksparrow see my comment in https://github.com/strongloop/loopback-next/pull/5403#discussion_r424718045
I think you are confused because you are new to this example module. Please note the doc is added in the README.md file of the example repo.
This example demos two scenarios: "mounts lb3 app to lb4" and "mounts lb4 including lb3 to express server"